src/components/SidebarMenu.vue

```diff
@@ -62,18 +62,20 @@
           @afterEnter="expandAfterEnter"
           @beforeLeave="expandBeforeLeave"
         >
-          <listItem
-            v-if="mobileItem && mobileItem.child"
-            :items="mobileItem.child"
-            :show-child="showChild"
-            :rtl="rtl"
-            :is-collapsed="isCollapsed"
-          >
-            <slot
-              slot="dropdown-icon"
-              name="dropdown-icon"
-            />
-          </listItem>
+          <div class="vsm--list">
+            <sidebar-menu-item
```

Comment:
loop through `mobileItem.child` and level 2 prop
```suggestion
            <sidebar-menu-item
              v-for="(item, index) in mobileItem.child"
              :key="index"
              :level="2"
```